test/built-ins/Proxy/deleteProperty/call-parameters.js

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2015 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 9.5.10
+description: >
+    [[Delete]] (P)
+
+    9. Let booleanTrapResult be ToBoolean(Call(trap, handler, «target, P»)).
+info: >
+    6.1.7.2 Object Internal Methods and Internal Slots
+
+    (...) Receiver is used as the this value when evaluating the code
+---*/
+
+var target = {
+    attr: 1
+};
+var handler = {
+    deleteProperty: function(t, prop) {
+        assert.sameValue(this, handler, "handler object as the trap context");
+        assert.sameValue(t, target, "first argument is the target object");
+        assert.sameValue(prop, "attr", "second argument is the property name");
+    }
+};
+var p = new Proxy(target, handler);
+
+delete p.attr;
```

Comment:
this is a regular property, I can't see how it would throw in strict mode. It's a configurable property, matching [12.5.4.2 ](https://people.mozilla.org/~jorendorff/es6-draft.html#sec-delete-operator-runtime-semantics-evaluation)